### DISCUSSION TITLE
Which battery powered knob do you use?

### DISCUSSION THREAD

**Discussion by andreapx** [Koenkk/zigbee2mqtt#25244] [Devices]
I need a battery powered knob to dimmer some lights, which one do you use?
I've seen [this one](https://www.aliexpress.com/item/1005007575864175.html?spm=a2g0o.productlist.main.27.3b8cTjsZTjsZJ6&algo_pvid=1b77936a-b972-44ef-9ed7-33e8fd9b4ad5&algo_exp_id=1b77936a-b972-44ef-9ed7-33e8fd9b4ad5-13&pdp_npi=4%40dis%21EUR%219.49%219.49%21%21%219.75%219.75%21%40211b612817344473806467011e9757%2112000041445354538%21sea%21IT%21108561050%21X&curPageLogUid=MsGucoZ4IyrM&utparam-url=scene%3Asearch%7Cquery_from%3A) that's pretty inexpensive, but it's not compatible with Zigbee2mqtt yet.
The Tuya ERS-10TZBVK-AA doesn't seem to be available anymore.
Thanks!

**Comment by djurny** [comment]
Hi there,
I have a very similar one, simple called "Smart Knob Dimmer" (TS004F _TZ3000_402vrq2i). Seems that inside they are all Tuya based, not sure what the differences are. Zigbee2MQTT treats most if not all of them as ERS-10TZBVK-AA.
In case you did not only mean _rotary_ knobs: For most of our house we use the IKEA Tradfri on/off buttons and some RODRET switches. Holding on/off can be interpreted as brightness up/down depending on how you set up your HASS automation.
Hope that helps.
Groetjes,

**Reply by andreapx** [reply]
> I have a very similar one, simple called "Smart Knob Dimmer" (TS004F _TZ3000_402vrq2i)
I can't find it, could you please send me a link?

> In case you did not only mean rotary knobs
I meant rotary knobs, sorry.
I've tried using the the hold function, but I'm not a real fan (I hate that method! 🙈😆)

**Reply by djurny** [reply]
Hi @andreapx, I got this one:
https://www.aliexpress.com/item/3256807534367914.html
Groetjes,

**Reply by andreapx** [reply]
Thanks!

**Comment by carrierone** [comment]
I just got one of these TS004F _TZ3000_402vrq2i rotary knobs from AliExpress and ended up here with Google naturally.

Z2M will add the device but I get "Not supported".  Exposes is empty.  See screenshots..  @djurny does yours work in Zigbee2MQTT?  Can you share your generated_external_definition?

![Screenshot 2025-01-10 212106](https://github.com/user-attachments/assets/4dcd183f-584f-438a-a718-5ed8d87a15bf)
![Screenshot 2025-01-10 212147](https://github.com/user-attachments/assets/b97b0812-040f-4a66-b2ce-66e4c2af3735)

**Reply by djurny** [reply]
Hi @carrierone,
The easiest way is to upgrade zigbee2mqtt to [v1.42](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.42.0).

If you cannot, the external definition I used is actually from @Velomeloman, who merged it into [v1.42](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.42.0).

See https://github.com/Koenkk/zigbee2mqtt/issues/23543#issuecomment-2508257550.

Groetjes,

**Reply by carrierone** [reply]
Oh great, thank you I'll give that a try! 🙂

**Reply by carrierone** [reply]
It does work great now!  but I am curious to know if you have experienced the same flaw with bind.   I did a bind to a group of lights and unbind on the controller so it can run the lights directly, and set it to command mode instead of event,  So the features of toggle and color temperature work fine however turning the knob sending brightness_step_down and brightness_step_up take the light group directly down to 1% and back up to 100% which is a problem.  Have you experienced the same?  I am thinking this could be a firmware bug or something.

**Reply by carrierone** [reply]
The action step size shows up as like 25-37 or so

**Reply by JeanneD4RK** [reply]
I could not achieve anything using binding. What options did you use in the bind ?
